src/main/java/com/simibubi/create/content/contraptions/relays/encased/EncasedShaftBlock.java

```diff
@@ -25,26 +25,18 @@
 import net.minecraft.world.phys.HitResult;
 
 public class EncasedShaftBlock extends AbstractEncasedShaftBlock
-	implements ITE<KineticTileEntity>, ISpecialBlockItemRequirement {
+	implements ITE<KineticTileEntity>, ISpecialBlockItemRequirement, IEncased {
 
-	private BlockEntry<CasingBlock> casing;
+	private Block casing;
 
-	public static EncasedShaftBlock andesite(Properties properties) {
-		return new EncasedShaftBlock(properties, AllBlocks.ANDESITE_CASING);
-	}
-
-	public static EncasedShaftBlock brass(Properties properties) {
-		return new EncasedShaftBlock(properties, AllBlocks.BRASS_CASING);
-	}
-
-	protected EncasedShaftBlock(Properties properties, BlockEntry<CasingBlock> casing) {
+	public EncasedShaftBlock(Properties properties) {
 		super(properties);
-		this.casing = casing;
 	}
 
-	public BlockEntry<CasingBlock> getCasing() {
+	public Block getCasing() {
 		return casing;
 	}
+	public void setCasing(Block casing) {this.casing = casing;}
```

Comment:
- Missing a new line before this method
- Missing new line after left bracket
- Missing new line before right bracket